test/cypress/integration/settings/your-psp-stripe-tasklist.cy.test.js

```diff
@@ -47,12 +53,14 @@ describe('Your PSP Stripe page', () => {
   beforeEach(() => {
     Cypress.Cookies.preserveOnce('session', 'gateway_account')
   })
+
   it('should contain Your PSP - Stripe heading', () => {
     setupYourPspStubs({})
     cy.setEncryptedCookies(userExternalId)
     cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
-    cy.get('.govuk-heading-m').should('contain', 'Information for Stripe')
+    cy.get('h1').should('contain', 'Information for Stripe')
   })
+
   it('should display all the required stripe tasks ', () => {
     setupYourPspStubs({})
```

Comment:
Instead of passing an `{}` - can we just leave it blank?
So put this instead:
`setupYourPspStubs()`